src/backend/nodes/copyfuncs.c

```diff
@@ -3134,6 +3134,7 @@ _copyQuery(const Query *from)
 	COPY_NODE_FIELD(constraintDeps);
 	COPY_NODE_FIELD(intoPolicy);
 	COPY_NODE_FIELD(intoClause);
+	COPY_SCALAR_FIELD(isCtas);
```

Comment:
If you add a variable in Query, you need to add similar code in more than this file/functions, maybe equalfuncs.c, outfast.c, outfuncs.c, readfast.c, readfuncs.c, etc.